test/pytest/conftest.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+import os
+import subprocess
+from typing import Iterable
+
+import pytest
+
+from cockpit._vendor.systemd_ctypes import EventLoopPolicy
+
+
+def my_subprocesses():
+    try:
+        return subprocess.check_output(['pgrep', '-a', '-P', f'{os.getpid()}'], universal_newlines=True).splitlines()
+    except subprocess.CalledProcessError:
+        # no processes found?  good!
+        return []
+
+
+def assert_no_subprocesses():
+    running = my_subprocesses()
+    if not running:
+        return
+
+    print('Some subprocesses still running', running)
```

Comment:
Format-wise this may look better in the `pytest.fail`, to avoid mangling stdout/err and breaking pytest's output format.

Reply:
Nevermind, the next commit already changed that again.